inspirehep/modules/migrator/cli.py

```diff
@@ -140,13 +140,13 @@ def get_collection(marc_record):
     results_length = results.count()
     with click.progressbar(results.yield_per(100), length=results_length) as bar:
         for obj in bar:
-            marc_record = marc_create_record(obj.marcxml, keep_singletons=False)
+            marc_record = create_record(obj.marcxml, keep_singletons=False)
```

Comment:
This is the only remaining use of upstream's `create_record`. We could avoid it, but since this function is going to die there's no point in making it nice.